index.js

```diff
@@ -4,10 +4,13 @@ const {
     setupAlwaysOnTopRender,
     setupAlwaysOnTopMain
 } = require('./alwaysontop');
+const { getWiFiStats, setupWiFiStats } = require('./wifistats');
 
 module.exports = {
+    getWiFiStats,
```

Comment:
Do you still intend to expose getWiFiStats for public use? I only ask because you didn't document it in the README and it seems like setupWiFiStats exposes a way to access getWiFiStats. Maybe the README should also mention how it sets getWiFiStats on a global.